provisioning/common/scripts/k8s/logs.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+if [ $# -ne 1 ]; then
+  echo "Please specify one argument: target directory"
+  exit 1
+fi
```

Comment:
Dumpuju sa logy vsetkych `pods` a vsetky `events`.
Nakoniec sa to nahraje do artefaktov buildu.